src/Symfony/Bundle/FrameworkBundle/Validator/ConstraintValidatorFactory.php

```diff
@@ -11,7 +11,7 @@
 
 namespace Symfony\Bundle\FrameworkBundle\Validator;
 
-use Symfony\Component\DependencyInjection\ContainerInterface;
+use Psr\Container\ContainerInterface;
```

Comment:
Note also that adding deprecations for using experimental features has been rejected in #21625.